src/main/java/project/ui/pages/MainPage.java

```diff
@@ -13,31 +13,74 @@
 package project.ui.pages;
 
 import core.utils.TrabajopolisUtils;
-import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import project.ui.BasePage;
 
 /**
  * MainPage class.
  *
  * @author John Salazar Pinto
  * @version 1.0
  */
-public class MainPage extends TrabajopolisUtils {
+public class MainPage extends BasePage {
+
```

Comment:
Please delete this unnecessary first line(according to your other classes).

Reply:
done